teuthology/task/install.py

```diff
@@ -1264,6 +1361,11 @@ def task(ctx, config):
     assert isinstance(config, dict), \
         "task install only supports a dictionary for configuration"
 
+    rhbuild=None
+    if config.get('rhbuild'):
+        rhbuild=config.get('rhbuild')
+        log.info("Build is %s " % rhbuild)
```

Comment:
This is the config dict (in yaml) that differentiates between rh testing and upstream. using this value the if statement above is taking different install approach.